packages/grid/src/features/standard-table/components/StandardTableHeadRow.tsx

```diff
@@ -52,7 +52,13 @@ export const StandardTableHeadRow = React.memo(function StandardTableHeadRow<
 
   return (
     <TableHeadRow
-      top={headerRowOffsetTop ?? 0}
+      top={
+        headerRowOffsetTop && groupConfigs
+          ? `calc(${headerRowOffsetTop} + 40px)`
```

Comment:
Where does 40px come from?

Reply:
You can do something similar here as in ColumnGroupRow to remove the extra condition.

Reply:
Fixed!